modules/migrations/git.go

```diff
@@ -53,17 +53,17 @@ func (g *PlainGitDownloader) GetReleases() ([]*base.Release, error) {
 	return nil, ErrNotSupported
 }
 
-// GetIssues returns issues according start and limit
-func (g *PlainGitDownloader) GetIssues(start, limit int) ([]*base.Issue, error) {
-	return nil, ErrNotSupported
+// GetIssues returns issues according page and PerPage
+func (g *PlainGitDownloader) GetIssues(page, PerPage int) ([]*base.Issue, bool, error) {
```

Comment:
`pageSize` would be better name imho
```suggestion
func (g *PlainGitDownloader) GetIssues(page, pageSize int) ([]*base.Issue, bool, error) {
```

Reply:
The name is from github's API document. :)

Reply:
Method argument then should be at least lowercase first letter